packages/documentation/docs/controls/_textarea-field_styleguide.mdx

```diff
@@ -0,0 +1,62 @@
+The Textarea component allows users to input multi-line text. It can be used in various forms where longer text input is required.
```

Comment:
We normally always add a “We typically use...” to the intro sentence. Like: The textarea component allows users to input multi-line text, making it ideal for forms that require longer text entries. We typically use textareas in scenarios such as feedback forms, comment sections, and message composition.

Also please pay attention to capitalization “textarea” vs “Textarea”.